flowgate-common/src/main/java/com/vmware/flowgate/common/MetricName.java

```diff
@@ -0,0 +1,17 @@
+package com.vmware.flowgate.common;
+
+public class MetricName {
+   public static final String TEMP = "TEMP";
+   public static final String HUMIDITY = "HUMIDITY";
+   public static final String AIRFLOW = "AIRFLOW";
```

Comment:
AIRFlOW is not a metric, since it used to represent the sensor type. but since we change to MetricName, so it's not correct to have it here.

Reply:
> AIRFlOW is not a metric, since it used to represent the sensor type. but since we change to MetricName, so it's not correct to have it here.

Removed